entrypoint.sh

```diff
@@ -1,5 +1,22 @@
-#!/bin/sh -l
+#!/bin/bash
+set -e
 
-echo "Start DX Scanner Action"
+if [[ $1 == *"bash" ]]; then
+  exec "/bin/bash"
+  exit 0;
+elif [[ $1 == *"sh" ]]; then
+  exec "/bin/sh"
+  exit 0;
+fi
 
-dx-scanner run --ci -r -a $1
+echo "Start DX Scanner Action"
+if [[ $1 == "dx-scanner "* || $1 == "dxs "* || $1 == "dxscanner "* ]]; then
+  echo $@
+  exec "$@"
+elif [[ $1 == "-"* ]]; then
+  echo dx-scanner run $@
+  exec dx-scanner run "$@"
```

Comment:
It's not possible to get help just for dx-scanner if I want to pass just `--help`. It runs help for dx-scanner run